apps/condo/domains/common/utils/filters.utils.ts

```diff
@@ -184,11 +185,20 @@ export function convertToOptions <T> (objects: T[], labelField: string, valueFie
 }
 
 export async function updateQuery (router: NextRouter, newFilters?: FiltersFromQueryType, sort?: string[], offset?: number): Promise<boolean> {
-    if (!offset && 'offset' in router.query)
+    if (!offset && 'offset' in router.query) {
         router.query['offset'] = '0'
+    }
+
+    const possibleFilters = pickBy(newFilters, (value, key) => !isEmpty(value))
+    const possibleQueryData = { ...router.query, sort, offset }
+    if (isEmpty(possibleFilters)) {
+        delete possibleQueryData['filters']
```

Comment:
maybe `possibleQueryData.filters` ?

Reply:
The `possibleQueryData` variable has no type, so I decided to use `['filters']` to not make the linter angry to me :)